### DISCUSSION TITLE
npx create-next-app@latest The syntax of the command is incorrect.

### DISCUSSION THREAD

**Discussion by ashan123345** [vercel/next.js#65815] [Help]
### Summary

When I tried to create Nextapp using the npx command, it kept saying," The syntax of the command is incorrect.
'"' is not recognised as an internal or external command,
operable program or batch file." I tried editing environment variables. I've tried to reinstall node js. Globally installing the next app, tired using yarn, but the issue is unresolved.
![Screenshot 2024-05-16 010940](https://github.com/vercel/next.js/assets/150546424/9dce7132-15d6-40fd-a7d6-ee99e29eab4b)

**Comment by jososadev** [comment]
Just tried and seems to be working fine.

  I'm using node `20.9.0`
  
  And running it like this `npx create-next-app`
  
  
<img width="583" alt="Screenshot 2024-05-15 at 2 09 02 PM" src="https://github.com/vercel/next.js/assets/157504690/496b698d-018c-43d9-9a7a-c1d80aaa1e0f">

**Reply by ashan123345** [reply]
it doesn't work I tried that.

**Comment by icyJoseph** [comment]
Hi,

You don't use the `@latest` when invoking the command. `npx` is a way to execute a binary, or program rather, but to install it, you have to `npm i -g create-next-app@latest`, for example
